package.json

```diff
@@ -2,7 +2,7 @@
   "name": "iotagent-ul",
```

Comment:
CNR file should be flushed in this PR (see https://github.com/telefonicaid/iotagent-ul/pull/424/files)